### ISSUE TITLE
The wide port-range declared by Che to be exposed is spewing errors on GCE

### ISSUE THREAD

**Issue by lazypower** [tengu-team/layer-eclipse-che#1]
I've validated this on the JAAS model, and a self hosted controller against GCE.

Relevant linked bugs:
https://bugs.launchpad.net/juju/+bug/1687838

We'll need to reduce the overall number of ports being open. To combat the issue I simply ran:

```
$ juju run --application eclipse-che "close-port 32768-65535"
```

Thus far this is only isolated to the GCE provider that I'm aware of.

**Comment by merlijn-sebrechts** [comment]
Hm, I'm not sure how to solve this. Eclipse Che uses these ports for the workspaces (Docker containers). Each workspace will use a number of ports in that range for ssh, http, etc.

https://eclipse.org/che/docs/setup/configuration/index.html#firewalls

**Comment by lazypower** [comment]
Yeah, I suspect the actual culprit behind this is the GCE provider in juju-core. However, without a fix on the horizon, I wanted to file this in both places to track the behavior in case someone else comes along and finds the issue on their own.

It's not obvious this is happening, as it just won't open the ports on GCE and you won't know this is the case unless you check your controller logs.

**Comment by merlijn-sebrechts** [comment]
Aight, thanks!
